From 450ed14271d6fe9ce2d04764084c90197b0e7abb Mon Sep 17 00:00:00 2001
From: maniacx <macs18max@gmail.com>
Date: Sat, 21 Nov 2015 19:10:09 +0400
Subject: [PATCH] hwcomposer: Add HWC_BLIT compositiontype value  - HWC_BLIT
 this layer will be handled in the HWC, using a blit engine  - setting this
 will inform Surfaceflinger to flip FB_TARGET

Change-Id: I13b0ea4c1b20e330a0b00c0ceef69cf54d06e8b6
---
 include/hardware/hwcomposer_defs.h | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/include/hardware/hwcomposer_defs.h b/include/hardware/hwcomposer_defs.h
index 9c7d790..bb3c737 100644
--- a/include/hardware/hwcomposer_defs.h
+++ b/include/hardware/hwcomposer_defs.h
@@ -116,8 +116,11 @@ enum {
        cursor overlay hardware. hwcomposer will also all async position updates
        of this layer outside of the normal prepare()/set() loop. Added in
        HWC_DEVICE_API_VERSION_1_4. */
-    HWC_CURSOR_OVERLAY =  5
- };
+    HWC_CURSOR_OVERLAY =  5,
+
+    /* this layer will be handled in the HWC, using a blit engine */
+    HWC_BLIT = 6,
+};
 /*
  * hwc_layer_t::blending values
  */
-- 
2.1.4

